synfig-studio/src/gui/instance.cpp

```diff
@@ -223,25 +223,128 @@ Instance::set_redo_status(bool x)
 	signal_undo_redo_status_changed()();
 }
 
+bool
+studio::Instance::run_plugin_with_arguments(std::string plugin_path, const std::vector<std::string>& args)
+{
+	OneMoment one_moment;
+
+	bool result = false;
+	String output;
+	String command = "";
+
+	// Path to python binary can be overridden
+	// with SYNFIG_PYTHON_BINARY env variable:
+	const char* custom_python_binary=getenv("SYNFIG_PYTHON_BINARY");
+	if(custom_python_binary) {
+		command=custom_python_binary;
+		if (!App::check_python_version(command)) {
+			output=_("Error: You need to have Python 3 installed.");
```

Comment:
This message is overriding by line 271 below. Besides, it doesn't make clear the searched python binary was from the env var.